#!/usr/bin/python3
def raise_exception():
    q  = 1 / "a"
